cocos/platform/android/libcocos2dx-with-controller/build.gradle

```diff
@@ -6,7 +6,7 @@ android {
 
     defaultConfig {
         minSdkVersion 10
-        targetSdkVersion 22
+        targetSdkVersion PROP_TARGET_SDK_VERSION
```

Comment:
are you sure ` targetSdkVersion android-13` is correct value for `targetSdkVersion` key in `defaultConfig`?